src/index.ts

```diff
@@ -9,15 +9,15 @@ export class FunctionsClient {
   constructor(
     url: string,
     {
-      headers = {},
+      headers,
```

Comment:
```suggestion
      headers = {},
```
For consistency I'd leave this.